### PR TITLE
Add Landman's spectral index fitting suite 'spimple'

### PR DESCRIPTION
**Pull request by Mulan-94**
Add cabs for the spectral index fitter, image convolver and primary beam -> power beam converter"

### CODE REVIEW

**Comment by Athanaseus** [comment]
ok to test

File: stimela/cargo/cab/spimple_binterp/parameters.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{
+    "task": "spimple imconv",
```
Comment (Athanaseus) [inline stimela/cargo/cab/spimple_binterp/parameters.json:2]: :s/imconv/binterp

**Review by Athanaseus — APPROVED** [review]
Just a minor comment.
Otherwise looks good.

File: stimela/cargo/cab/spimple_binterp/parameters.json (hunk 1 above)
Comment (Mulan-94) [reply stimela/cargo/cab/spimple_binterp/parameters.json]: this is resolved

**Comment by Athanaseus** [comment]
Test failure:
``` Error response from daemon: manifest for quay.io/stimela/spimple:1.7.12b not found: manifest unknown: manifest unknown```
Is the image uploaded to quay or specified correctly?

**Comment by o-smirnov** [comment]
@Mulan-94 could you take another look please? There's a PR on Caracal that's depending on this.

**Comment by SpheMakh** [comment]
Pinging @Mulan-94

**Pull request merged**